### ISSUE TITLE
Training Issues

### ISSUE THREAD

**Issue by MZMMSEC** [Michel-liu/FatFormer#7]
Hi，

This is an inspiring work, and I am studying it now! Since you do not release the training code, I have to train it myself based on your current released codes! and Specifically, I used my dataset. However, the training accuracy is always around 50%, though the loss decreases. 

I have several questions about training:

1. Can I build the model directly using the model code in your ``main.py``? as follows. Is there anything/any place that needs to be modified on the model implementation codes of your GitHub repo to adapt to the training process?
![1725458912772](https://github.com/user-attachments/assets/8bd3778d-2ea1-4a89-8573-4772dbec1afc)

2. Is it correct that my training code implementations, as follows, where the ``criterion`` function is ``nn.CrossEntropyLoss()``?
![1725458786706](https://github.com/user-attachments/assets/e9b3f622-9c75-4e8a-9d5c-9c8f9512996a)

Thanks for your attention!
Best

**Comment by Tuesdayo** [comment]
hi! Have you reproduced the results in paper now? My accuracy still stays at 50%.
